main/tests/server/src/com/google/refine/expr/functions/arrays/ZipTests.java

```diff
@@ -0,0 +1,48 @@
+
+package com.google.refine.expr.functions.arrays;
+
+import org.testng.annotations.Test;
+
+import com.google.refine.expr.ParsingException;
+import com.google.refine.grel.GrelTestBase;
+
+public class ZipTests extends GrelTestBase {
+
+    @Test
+    public void zipArray() throws ParsingException {
+        String[] test = { "[ [1,2,3], ['A','B','C'] ].zip()", "[[1, A], [2, B], [3, C]]" };
```

Comment:
The tests need to be updated to the following form which matches the original specification
```suggestion
        String[] test = { "zip([1,2,3],['A','B','C'])", "[[1, A], [2, B], [3, C]]" };
```

Reply:
Updated all test cases